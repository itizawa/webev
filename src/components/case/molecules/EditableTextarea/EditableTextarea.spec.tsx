import renderer from 'react-test-renderer';
import { EditableTextarea } from './index';

test('EditableTextarea', () => {
  const tree = renderer.create(<EditableTextarea onChange={expect.any(Function)} />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('EditableTextarea is header', () => {
  const tree = renderer.create(<EditableTextarea onChange={expect.any(Function)} isHeader />).toJSON();

  expect(tree).toMatchSnapshot();
});
