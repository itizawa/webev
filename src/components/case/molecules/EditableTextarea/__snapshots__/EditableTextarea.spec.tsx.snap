// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditableTextarea 1`] = `
.c0 {
  background: transparent;
  border: none;
  -ms-overflow-style: none;
  -webkit-scrollbar-width: none;
  -moz-scrollbar-width: none;
  -ms-scrollbar-width: none;
  scrollbar-width: none;
}

.c0::-webkit-scrollbar {
  display: none;
}

.c0:hover {
  background: #232323;
}

.c0:hover::-webkit-input-placeholder {
  color: #ccc;
}

.c0:hover::-moz-placeholder {
  color: #ccc;
}

.c0:hover:-ms-input-placeholder {
  color: #ccc;
}

.c0:hover::placeholder {
  color: #ccc;
}

.c0:focus {
  background: transparent;
}

.c0:focus::-webkit-input-placeholder {
  color: #ccc;
}

.c0:focus::-moz-placeholder {
  color: #ccc;
}

.c0:focus:-ms-input-placeholder {
  color: #ccc;
}

.c0:focus::placeholder {
  color: #ccc;
}

<textarea
  className="c0 form-control text-white text-nowrap overflow-scroll  pb-md-0 w-100"
  onChange={[Function]}
  placeholder=""
  value=""
/>
`;

exports[`EditableTextarea is header 1`] = `
.c0 {
  background: transparent;
  border: none;
  -ms-overflow-style: none;
  -webkit-scrollbar-width: none;
  -moz-scrollbar-width: none;
  -ms-scrollbar-width: none;
  scrollbar-width: none;
}

.c0::-webkit-scrollbar {
  display: none;
}

.c0:hover {
  background: #232323;
}

.c0:hover::-webkit-input-placeholder {
  color: #ccc;
}

.c0:hover::-moz-placeholder {
  color: #ccc;
}

.c0:hover:-ms-input-placeholder {
  color: #ccc;
}

.c0:hover::placeholder {
  color: #ccc;
}

.c0:focus {
  background: transparent;
}

.c0:focus::-webkit-input-placeholder {
  color: #ccc;
}

.c0:focus::-moz-placeholder {
  color: #ccc;
}

.c0:focus:-ms-input-placeholder {
  color: #ccc;
}

.c0:focus::placeholder {
  color: #ccc;
}

<textarea
  className="c0 form-control text-white text-nowrap overflow-scroll fs-1 pb-md-0 w-100"
  onChange={[Function]}
  placeholder=""
  value=""
/>
`;
